jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/RetryRequestException.java

```diff
@@ -0,0 +1,26 @@
+//
+// ========================================================================
+// Copyright (c) 1995 Mort Bay Consulting Pty Ltd and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+// which is available at https://www.apache.org/licenses/LICENSE-2.0.
+//
+// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+// ========================================================================
+//
+
+package org.eclipse.jetty.client;
+
+/**
+ * <p>Exception indicating that the HTTP request was not handled
+ * by the server, and can therefore be retried, if possible.</p>
+ */
+public class RetryRequestException extends RuntimeException
```

Comment:
This doesn't need to be a `RuntimeException`, so it could extend from `Exception`.